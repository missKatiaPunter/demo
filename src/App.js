import './App.css';
import Counter from './components/Counter';

function App() {
  return (
    <div className="App">
      <p>The fantastic counter!</p>
      <Counter />
    </div>
  );
}

export default App;
