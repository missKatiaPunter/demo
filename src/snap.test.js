
import React from 'react';

import renderer from 'react-test-renderer';
import App from './App';

test("HomeScreen renders consistently", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
});

