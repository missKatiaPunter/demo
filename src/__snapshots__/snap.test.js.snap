// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders consistently 1`] = `
<div
  className="App"
>
  <p>
    The fantastic counter app
  </p>
  <div>
    <button
      data-testid="counter-button"
      onClick={[Function]}
    >
      0
    </button>
  </div>
</div>
`;
